tools/ci_build/github/azure-pipelines/templates/explicitly-defined-final-tasks.yml

```diff
@@ -21,7 +21,7 @@ steps:
   condition: eq(variables['Agent.OS'], 'Linux')
   continueOnError: true
 
-- script: rm -rf /tmp/*
+- script: sudo rm -rf /tmp/*
```

Comment:
Why do you need sudo?

Reply:
without sudo, the /tmp couldn't be clean.
https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=903464&view=logs&j=7536d2cd-87d4-54fe-4891-bfbbf2741d83&t=5bc73d64-a4be-5fce-feb0-f7691586ef68

Reply:
But this folder contains system files of the build machine! Why do you need to delete them all? This "rm -rf " runs outside of a docker container.

Reply:
To save space, the post-job of cache task will tar the image file and upload it, which needs a lot of disk space.
Let me try removing the step, maybe it's not necessary now.

Reply:
Put your temp files in $(Agent.TempDirectory), which is cleaned after each pipeline job. See: https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml

Reply:
I removed that step. So far, the space is enough as the agent is stateless.